### PR TITLE
Update TRACONBoundaries.geojson Update UNWW_APP, UNEE_APP

### PR DESCRIPTION
**Pull request by ikv20009**

### CODE REVIEW

File: TRACONBoundaries.geojson [hunk 1]
```diff
@@ -502,12 +502,11 @@
...
-    {"type":"Feature","properties":{"id":"UNEE","prefix":["UNEE"],"name":"Kemerovo Approach"},"geometry":{"type":"MultiPolygon","coordinates":[[[[89.08306,56.06667],[86.00000,56.8306],[85.06667,55.80000],[85.20000,55.71667],[85.20000,55.50000],[84.70000,55.23306],[84.66667,54.83306],[84.70000,54.56667],[85.48306,54.35000],[85.75000,54.45000],[87.50000,54.53306],[88.66806,55.57667],[88.97778,55.69889],[89.08306,56.06667]]]]}},
-    {"type":"Feature","properties":{"id":"UNNT","prefix":["UNNT"],"name":"Novosibirsk Approach"},"geometry":{"type":"MultiPolygon","coordinates":[[[[82.26167,56.02],[81.86667,55.91667],[81.585,55.79333],[81.25,55.61667],[81.135,55.46833],[81.01833,55.32333],[80.99167,55.215],[80.93333,54.93333],[80.96667,54.825],[81.02833,54.60667],[81.01667,54.43333],[81.73167,54.02667],[82.78333,54.02667],[83.40333,54.12167],[83.98333,54.06333],[84.045,54.24167],[84.13333,54.5],[84.15,54.53333],[84.26667,54.67833],[84.3,54.71667],[84.66667,54.83333],[84.7,55.23333],[85.2,55.5],[85.2,55.71667],[85.06667,55.8],[84.66667,56.05],[84.25,56],[83.8,56.01667],[83.61667,56.1],[83.2,56.18333],[82.63333,56.08333],[82.52333,56.06667],[82.44,56.05833],[82.26167,56.02]]]]}},
```
Comment (kmoberg) [inline TRACONBoundaries.geojson:506]: Is this line removal intentional?

**Review by kmoberg — CHANGES_REQUESTED** [review]

File: TRACONBoundaries.geojson (hunk 1 above)
Comment (ikv20009) [reply TRACONBoundaries.geojson:506]: Yes, updating borders.

Comment (ikv20009) [reply TRACONBoundaries.geojson:506]: Delete UNWW and UNEE

Comment (ikv20009) [reply TRACONBoundaries.geojson:506]: Update UNWW, UNEE

**Pull request closed without merging**